example/install_rackhd.sh

```diff
@@ -0,0 +1,236 @@
+#!/bin/bash
+#
+########################################################################################################
+#
+#  this script is based on RackHD document "Ubuntu Package Based Installation"
+#
+#  http://rackhd.readthedocs.io/en/latest/rackhd2.0/ubuntu_package_installation.html?highlight=install
```

Comment:
@panpan0000 we are moving away from separate 1.1 / 2.0 user's guide, the link will no longer valid.  Check out: https://github.com/RackHD/docs/pull/312#issuecomment-235677275 and the latest comment "Just added a commit that deletes the RackHD 2.0 user's guide and adds 2.0 API commands to the existing User's Guide when there's differences"